packages/1155-contracts/src/delegation/ZoraCreator1155Attribution.sol

```diff
@@ -219,7 +336,134 @@ library PremintTokenSetup {
                 saleStart: saleStart,
                 saleEnd: saleEnd,
                 maxTokensPerAddress: maxTokensPerAddress,
-                fundsRecipient: creator
+                fundsRecipient: address(0)
```

Comment:
confim with joe regarding how this should be set (with field called `paidMintFundsRecipient`)